lib/semantics/check-do-concurrent.cc

```diff
@@ -160,25 +192,38 @@ class DoConcurrentEnforcement {
     }
     return false;
   }
+  bool addName(const std::optional<parser::Name> &nm) {
+    if (nm.has_value())
+      names_.insert(nm.value().ToString());
+    return true;
+  }
 
+  bool inDoConcurrent{false};
```

Comment:
We conventionally append an underscore `_` to the names of private data members in classes.

Reply:
This was a stray variable. I removed it.